gatsby-config.js

```diff
@@ -111,6 +111,16 @@ module.exports = {
       },
     },
     `gatsby-plugin-sitemap`,
+    ...(!isDeployPreview
```

Comment:
This would result in the plugin running during local development. Will Plausible have any issue with that?

Reply:
I ran it locally, and checking the dev tools, nothing is being shown regarding Plausible, so it seems we're in the clear. @Integrayshaun can you confirm if anything is being tracked

Reply:
Plausible wont, but I'm sure that we don't want localhost records in our analytics. Fixed